app/components/auto_tag_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AutoTagComponent < ViewComponent::Base
+  def initialize(text:)
+    colours = {
+      "to do": "grey",
+      "cannot start yet": "grey",
+      "setup complete": "green",
+      "done": "green",
```

Comment:
These look like the wrong colours to me. Please reference the prototype, not the wireframes, as the source of truth

Reply:
They're not all on the prototype (at least, the school preparing to do cip+fip prototypes), but I changed the one I could see which was wrong